store/sqlstore/upgrade.go

```diff
@@ -1310,7 +1310,34 @@ func upgradeDatabaseToVersion600(sqlStore *SqlStore) {
 	sqlStore.AlterColumnTypeIfExists("Users", "Props", "JSON", "jsonb")
 	sqlStore.AlterColumnTypeIfExists("Users", "NotifyProps", "JSON", "jsonb")
 	sqlStore.AlterColumnTypeIfExists("Users", "Timezone", "JSON", "jsonb")
-
+	fixCRTChannelCountsAndUnreadsUpdateColumn(sqlStore)
 	// saveSchemaVersion(sqlStore, Version600)
 	// }
 }
+
+func fixCRTChannelCountsAndUnreadsUpdateColumn(sqlStore *SqlStore) {
+	created := sqlStore.CreateColumnIfNotExists("ChannelMembers", "CRTFixDone", "tinyint(1)", "boolean", "1")
```

Comment:
Newly created channel memberships don't have this problem so the default is `true`.

Reply:
On my test DB (docker mysql 5.7.12) containing 4.2 million channel members, this takes about 3 minutes.